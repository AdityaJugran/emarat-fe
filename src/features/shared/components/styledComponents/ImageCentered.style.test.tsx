import renderer from 'react-test-renderer';
import ImageCenetered from 'features/shared/components/styledComponents/ImageCentered.style';

it('Styled Title Snapshot', () => {
	const tree = renderer.create(<ImageCenetered />).toJSON();
	expect(tree).toMatchSnapshot();
});
