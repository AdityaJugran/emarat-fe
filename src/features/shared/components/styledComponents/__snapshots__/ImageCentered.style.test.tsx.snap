// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styled Title Snapshot 1`] = `
<img
  className="sc-bdfBQB fTcTSm"
/>
`;
